packages/react/src/pages/analytics-dashboard/analytics-dashboard.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import Toolbar, { Item } from 'devextreme-react/toolbar';
+import Button from 'devextreme-react/button';
+import { getOpportunitiesByCategory, getSalesByCategory, getSales, getSalesByStateAndCity, getSalesByState } from 'dx-rwa-data';
+import './analytics-dashboard.scss';
+import { RevenueSnapshotCard } from './cards/RevenueSnapshotCard';
+import { RevenueAnalysisCard } from './cards/RevenueAnalysisCard';
+import { ConversionFunnelCard } from './cards/ConversionFunnelCard';
+import { RevenueCard } from './cards/RevenueCard';
+import { ConversionTicker, LeadsTicker, OpportunitiesTicker, RevenueTotalTicker } from './cards/TickerCards';
+
+const tabItems = {
+  Week: {
+    period: '2020-01-24/2020-01-31',
+    index: 0,
+  },
+  '2 Weeks': {
+    period: '2020-01-14/2020-01-31',
+    index: 1,
+  },
+  Month: {
+    period: '2020-01-01/2020-02-01',
+    index: 2,
+  },
+  Year: {
+    period: '2020-01-01/2021-01-01',
+    index: 3,
+  },
+  All: {
+    period: '2018-01-01/2022-01-01',
+    index: 4,
+  },
+};
+
+const DEFAULT_TAB_KEY = 'All';
+
+const calculateTotal = (data: Array<any>) => {
```

Comment:
Can we use different type there, not any?

Reply:
Taken from angular project as is. Data layer functions are untyped, total is calculated from their result values. Removed type for now, it is better to make types for all data layer functions later.